Library/Formula/slacknimate.rb

```diff
@@ -0,0 +1,24 @@
+class Slacknimate < Formula
+  desc "text animation for Slack messages"
+  homepage "https://github.com/mroth/slacknimate"
+  url "https://github.com/mroth/slacknimate/archive/v1.0.0.tar.gz"
+  sha256 "7d495e2233a379f3ffffb2445f4b41b4306e52049d71f4c4c873786bb4048f45"
+  head "https://github.com/mroth/slacknimate.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/mroth"
+    ln_s buildpath, buildpath/"src/github.com/mroth/slacknimate"
+    system "glide", "install"
```

Comment:
Gotcha, thanks @bfontaine.  I was not previously aware of the design goal for homebrew of having the `install` phase not require any additional network resources.  This approach would certainly violate that.

For my projects, I tend to find reproducible builds / overall end-user experience is more important than any inconvenience we as the upstream developers face, so my plan is to to make the modifications to the upstream project so that `vendor/` directory is checked into version control directly, and then cut a new release version here.  I'll update this PR when that happens (hopefully later today, depending on other workload).

(As a sidenote, is that TODO around build-dep caching documented somewhere public?  I'd be potentially interested in getting involved and helping out.)